card/card.html

```diff
@@ -0,0 +1,164 @@
+
+               <div class="card text-end">
+                  <div class="card-body">
+                     <h5 class="card-title">Special title treatment</h5>
+                     <p class="card-text">
+                        With supporting text below as a natural lead-in to
+                        additional content.
+                     </p>
+                     <button class="btn btn-primary">Go somewhere</button>
+                  </div>
+               </div>
+            </section>
+
+            <iframe
+               src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=sharp&l=htmlmixed&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522card%2522%253E%250A%2520%2520%253Cdiv%2520class%253D%2522card-body%2522%253E%250A%2520%2520%2520%2520%253Ch5%2520class%253D%2522card-title%2522%253ESpecial%2520title%2520treatment%253C%252Fh5%253E%250A%2520%2520%2520%2520%253Cp%2520class%253D%2522card-text%2522%253EWith%2520supporting%2520text%2520below%2520as%2520a%2520natural%2520lead-in%2520to%2520additional%2520content.%253C%252Fp%253E%250A%2520%2520%2520%2520%253Ca%2520href%253D%2522%2523%2522%2520class%253D%2522btn%2520btn-primary%2522%253EGo%2520somewhere%253C%252Fa%253E%250A%2520%2520%253C%252Fdiv%253E%250A%253C%252Fdiv%253E%250A%250A%253Cdiv%2520class%253D%2522card%2520text-center%2522%253E%250A%2520%2520%253Cdiv%2520class%253D%2522card-body%2522%253E%250A%2520%2520%2520%2520%253Ch5%2520class%253D%2522card-title%2522%253ESpecial%2520title%2520treatment%253C%252Fh5%253E%250A%2520%2520%2520%2520%253Cp%2520class%253D%2522card-text%2522%253EWith%2520supporting%2520text%2520below%2520as%2520a%2520natural%2520lead-in%2520to%2520additional%2520content.%253C%252Fp%253E%250A%2520%2520%2520%2520%253Ca%2520href%253D%2522%2523%2522%2520class%253D%2522btn%2520btn-primary%2522%253EGo%2520somewhere%253C%252Fa%253E%250A%2520%2520%253C%252Fdiv%253E%250A%253C%252Fdiv%253E%250A%250A%253Cdiv%2520class%253D%2522card%2520text-end%2522%253E%250A%2520%2520%253Cdiv%2520class%253D%2522card-body%2522%253E%250A%2520%2520%2520%2520%253Ch5%2520class%253D%2522card-title%2522%253ESpecial%2520title%2520treatment%253C%252Fh5%253E%250A%2520%2520%2520%2520%253Cp%2520class%253D%2522card-text%2522%253EWith%2520supporting%2520text%2520below%2520as%2520a%2520natural%2520lead-in%2520to%2520additional%2520content.%253C%252Fp%253E%250A%2520%2520%2520%2520%253Ca%2520href%253D%2522%2523%2522%2520class%253D%2522btn%2520btn-primary%2522%253EGo%2520somewhere%253C%252Fa%253E%250A%2520%2520%253C%252Fdiv%253E%250A%253C%252Fdiv%253E"
+               style="
+                  width: 872px;
+                  height: 488px;
+                  border: 0;
+                  transform: scale(1);
+                  overflow: hidden;
+               "
+               sandbox="allow-scripts allow-same-origin"
+            >
+            </iframe>
```

Comment:
This iframe is scaling horizontally and hence giving a horizontal scroll bar.
Fix it for consistent UI design.